packages/web/src/scss/components/UNSTABLE_Toggle/_UNSTABLE_Toggle.scss

```diff
@@ -38,48 +38,97 @@ $_field-name: 'UNSTABLE_Toggle';
 .UNSTABLE_Toggle__input {
     @include form-fields-tools.inline-field-input();
 
+    display: grid;
     flex-shrink: 0;
+    grid-template-areas: 'toggle';
     width: theme.$input-width;
     height: theme.$input-height;
-    border: theme.$input-border-width solid form-fields-theme.$inline-field-input-color-unchecked;
+    border: theme.$input-border-width solid theme.$input-border-color;
     border-radius: theme.$input-border-radius;
-    background-image: svg.escape(theme.$input-mark);
-    background-position-x: theme.$input-mark-position-x;
-    background-position-y: theme.$input-mark-position-y;
-    background-size: theme.$input-mark-width theme.$input-mark-height;
-    background-repeat: no-repeat;
-    background-color: form-fields-theme.$inline-field-input-color-unchecked;
+    background-color: theme.$input-background-color;
 
     @media (prefers-reduced-motion: no-preference) {
-        transition-property: border-color, background-color, background-position;
+        transition-property: border-color, background-color;
         transition-duration: theme.$input-transition-duration;
         transition-timing-function: theme.$input-transition-timing;
     }
 
+    &::before,
+    &::after {
+        content: '';
+        display: block;
+        grid-area: toggle;
+        width: theme.$input-mark-size;
+        height: theme.$input-mark-size;
+        margin: 1px;
+        border-radius: theme.$input-mark-border-radius;
+        background-image: svg.escape(theme.$input-mark);
```

Comment:
Isn't `url()` missing here? 🤔 It shouldn't be part of the `ecape()` function. Alternatively, we could have another function like `escaped-url()`.

Reply:
`url()` is part of the SCSS variable.